Transformations/utilities/testTimestamp.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
```

Comment:
Add comments explaining the purpose.

Reply:
updated documentation in [readme](https://github.com/VCityTeam/UD-Graph/blob/rules/Transformations/utilities/README.md#addtimestamps)